dotnet/README.md

```diff
@@ -121,31 +121,29 @@ more features, such as connectors to services and Skills to perform specific act
 
 This meta package includes core packages and OpenAI connectors, allowing to run
 most samples and build apps with OpenAI and Azure OpenAI.
 
 Packages included in **Microsoft.SemanticKernel**:
 
-1. **Microsoft.SemanticKernel.Abstractions**: contains common interfaces and classes
+1. [Microsoft.SemanticKernel.Abstractions](https://www.nuget.org/packages/Microsoft.SemanticKernel.Abstractions/): contains common interfaces and classes
   used by the core and other SK components.
-1. **Microsoft.SemanticKernel.Core**: contains the core logic of SK, such as prompt
+1. [Microsoft.SemanticKernel.Core](https://www.nuget.org/packages/Microsoft.SemanticKernel.Core/): contains the core logic of SK, such as prompt
   engineering, semantic memory and semantic functions definition and orchestration.
-1. **Microsoft.SemanticKernel.Connectors.AI.OpenAI**: connectors to OpenAI and Azure
-  OpenAI, allowing to run semantic functions, chats, image generation with GPT3,
-  GPT3.5, GPT4, DALL-E2. Includes also GPT tokenizers.
+1. [Microsoft.SemanticKernel.Connectors.AI.OpenAI](https://www.nuget.org/packages/Microsoft.SemanticKernel.Connectors.AI.OpenAI/): connectors to OpenAI and Azure
+  OpenAI, allowing to run semantic functions, chats, image generation with GPT3, GPT3.5, GPT4, DALL-E2. Includes also GPT tokenizers.
 
 Other SK packages available at nuget.org:
 
-1. **Microsoft.SemanticKernel.Connectors.Memory.Qdrant**: Qdrant connector for
+1. [Microsoft.SemanticKernel.Connectors.Memory.Qdrant](https://www.nuget.org/packages/Microsoft.SemanticKernel.Connectors.Memory.Qdrant): Qdrant connector for
    skills and semantic memory.
-2. **Microsoft.SemanticKernel.Connectors.Memory.Sqlite**: SQLite connector for
+2. [Microsoft.SemanticKernel.Connectors.Memory.Sqlite](https://www.nuget.org/packages/Microsoft.SemanticKernel.Connectors.Memory.Sqlite): SQLite connector for
    skills and semantic memory
-3. **Microsoft.SemanticKernel.Skills.Document**: Document Skill: Word processing,
+3. [Microsoft.SemanticKernel.Skills.Document](https://www.nuget.org/packages/Microsoft.SemanticKernel.Skills.Document): Document Skill: Word processing,
    OpenXML, etc.
-4. **Microsoft.SemanticKernel.Skills.MsGraph**: Microsoft Graph Skill: access your
+4. [Microsoft.SemanticKernel.Skills.MsGraph](https://www.nuget.org/packages/Microsoft.SemanticKernel.Skills.MsGraph/): Microsoft Graph Skill: access your
    tenant data, schedule meetings, send emails, etc.
-5. **Microsoft.SemanticKernel.Skills.OpenAPI**: OpenAPI skill.
-6. **Microsoft.SemanticKernel.Skills.Web**: Web Skill: search the web, download
-   files, etc.
\ No newline at end of file
+5. [Microsoft.SemanticKernel.Skills.OpenAPI](https://www.nuget.org/packages/Microsoft.SemanticKernel.Skills.OpenAPI): OpenAPI skill.
+6. [Microsoft.SemanticKernel.Skills.Web](https://www.nuget.org/packages/Microsoft.SemanticKernel.Skills.Web/): Web Skill: search the web, download files, etc.
```

Comment:
Note, we now have a LOT more packages than are listed here.
https://www.nuget.org/packages?q=Microsoft.SemanticKernel

@matthewbolanos FYI we may want to flesh out this list on the readme. Since this is quite dynamic with new packages all the time, we may want to write an Action script to update this list periodically from the nuget feed, or at least alert us to missing entries.